docs/sources/administration/data-source-management/teamlbac/_index.md

```diff
@@ -14,71 +14,55 @@ weight: 100
 
 # Team LBAC
 
-{{% admonition type="note" %}}
-Creating Team LBAC rules is available for preview for logs with Loki in Grafana Cloud. Report any unexpected behavior to the Grafana Support team.
-{{% /admonition %}}
+Team Label Based Access Control (LBAC) simplifies and streamlines data source access management based on team memberships.
 
-**Current Limitation:**
+You can configure user access based upon team memberships using LogQL.
+Team LBAC controls access to logs depending on the rules set for each team.
 
-- Any user with `query` permissions for a Loki data source can query all logs if there are no Team LBAC rules configured for any of the users team.
-- An admin that is part of a team, would have it's Team LBAC rules applied to the request.
-- Team LBAC rules will not be applied if the linked Cloud Access Policy has label selectors.
+This feature addresses two common challenges faced by Grafana users:
 
-Grafana's new **Team LBAC** (Label Based Access Control) feature for Loki is a significant enhancement that simplifies and streamlines data source access management based on team memberships.
+1. Having a high number of Grafana Cloud data sources.
+   Team LBAC lets Grafana administrators reduce the total number of data sources per instance from hundreds, to one.
+1. Using the same dashboard across multiple teams.
+   Team LBAC lets Grafana Teams use the same dashboard with different access control rules.
 
-**Team LBAC** in the context of Loki, is a way to control access to logs based on labels present depending on the rules set for each team. Users wanting fine grained access to their logs in Loki, can now configure their users access based on their team memberships via **LogQL**.
+To set up Team LBAC for a Loki data source, refer to [Configure Team LBAC](https://grafana.com/docs/grafana/<GRAFANA_VERSION>/administration/data-source-management/teamlbac/configure-teamlbac-for-loki/).
 
-This feature addresses two common challenge faced by Grafana users:
+{{< admonition type="note" >}}
```

Comment:
Reviewing this patch, I think that I should probably put this back at the top of the page